python/pylibcudf/pylibcudf/io/csv.pyi

```diff
@@ -54,7 +96,7 @@ def read_csv(
     # detect_whitespace_around_quotes: bool = False,
     # timestamp_type: DataType = DataType(type_id.EMPTY),
 ) -> TableWithMetadata: ...
-def write_csv(options: CsvWriterOptionsBuilder) -> None: ...
```

Comment:
Was this annotation incorrect before?

Reply:
No I don't think so. I wanted to stay consistent because I noticed we dropped the `None` in other PRs.

Reply:
Ah I see. IMO we should still include `None` as the output type cc @wence-

Reply:
I'll follow up based on what we decide